### ISSUE TITLE
"See more" links on the homepage are not accessible

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#3633]
If a user of assistive tech skips to a link that says "see more" it will provide no context as to what that link represents. We should add the context even if it's visually-hidden, though better would be to restore the original link text that made it clear for all users.

**Comment by willdurand** [comment]
Yeah I dont know why this has changed to be honest, maybe @devaneymoz or @pwalm could provide context :)

**Comment by devaneymoz** [comment]
Here's new link copy, as ordered on the homepage...

See more popular extensions

See more tab extensions

See more up & coming extensions

See more featured themes

**Comment by AlexandraMoga** [comment]
Verified on AMO-dev with FF56 (Win10 and Android 7.0)

![image](https://user-images.githubusercontent.com/31961530/32316201-eb568bc2-bfb7-11e7-931e-bafa5ad1de81.png)
![image](https://user-images.githubusercontent.com/31961530/32316283-388c00ca-bfb8-11e7-84a6-c8d163ded96c.png)

@willdurand Shouldn't the 'See more...' links be more emphasized, for example, showing a bottom border on hover? This will make them more consistent with other links in the page (see Header or Footer).

**Comment by willdurand** [comment]
@AlexandraMoga indeed, category pages should have new links too. Could you please open a separate issue for this?

**Comment by willdurand** [comment]
> Shouldn't the 'See more...' links be more emphasized, for example, showing a bottom border on hover? This will make them more consistent with other links in the page (see Header or Footer).

I dont think we have a special style for that unfortunately.

**Comment by AlexandraMoga** [comment]
> I dont think we have a special style for that unfortunately.

Maybe this is something to be taken into consideration for future design changes :)
